src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -557,6 +557,15 @@
   - name: --enable-vtpm
     type: bool
     short-summary: Enable vTPM on all node pools in the cluster. Must use VMSS agent pool type.
+  - name: --enable-acns
+    type: bool
+    short-summary: Enable advanced network functionalities on a cluster. Enabling this will incur additional costs.
+  - name: -disable-acns-observability
+    type: bool
+    short-summary: Used to disable advanced networking observability features on a clusters when enabling advanced networking features with "--enable-acns".
+  - name: --disable-acns-security
+    type: bool
+    short-summary: Used to disable advanced networking observability features on a clusters when enabling advanced networking features with "--enable-acns".
```

Comment:
change to `Used to disable advanced networking security features on a clusters when enabling advanced networking features with "--enable-acns".` as well as in update